doc/nrf/libraries/caf/sensor_data_aggregator.rst

```diff
@@ -0,0 +1,79 @@
+.. _caf_sensor_data_aggregator:
+
+CAF: Sensor data aggregator module
+##################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The |sensor_data_aggregator| of the :ref:`lib_caf` (CAF) is a simple module responsible for aggregating sensor data in form of ``sensor_event`` and passing them further in packages.
+It can be used in multi-core chips to reduce power consumption.
+In this case, one core gathers data from sensors and when there is sufficient data to analyze, the first core wakes up the other core and sends the aggregated data to that core.
```

Comment:
- "In this case" -- are there other use cases besides the multi-core one?
- "when there is sufficient data to analyze, the first core wakes up the other core and sends the aggregated data to that core." -- in line 22, you use an "if" for this option, so my question is: is this optional? what is the condition for if? It's a bit confusing now, as this is presented here as the default option, not an optional one.

Reply:
Yes, you ca use it also in single core device. The if is if you use two core. In my opinion "If you want to receive aggregated data on another core," is quite clear about that.